src/app/common/clarity-utils.ts

```diff
@@ -0,0 +1,49 @@
+import { ClarityType, ClarityValue, cvToString } from '@stacks/transactions';
+import { principalToString } from '@stacks/transactions/dist/esm/clarity/types/principalCV';
+
+export function clarityValueToDisplay(
```

Comment:
What are the differences between this function, and the similar `cvToString` in Stacks.js?

Reply:
This one is to display the value with indentation instead of parentheses like in the figma design spec. 
See https://github.com/hirosystems/stacks-wallet-web/pull/2405#issue-1227801915